README.md

```diff
@@ -1,21 +1,23 @@
-# hello-bootstrap-rails
+# LaunchDarkly sample Rails application with bootstrapping
 
-A Rails app demonstrating LaunchDarkly server-side bootstrapping.
+We've built a simple Rails app that demonstrates LaunchDarkly bootstrapping.
 
-The following files were modified and/or added to create this example
+We modified and/or added the following files to create this example:
 - app/views/hello_world/index.html.erb
 - app/controllers/hello_world_controller.rb
 - config/puma.rb
 
-The rest of the files were created by rails and are unmodified.
+Rails created the rest of the files and are unmodified.
 
-## Running Locally
+Below, you'll find the build procedure. For more comprehensive instructions, you can visit your [Quickstart page](https://app.launchdarkly.com/quickstart#/) or the [Ruby reference guide](https://docs.launchdarkly.com/sdk/server-side/ruby).
 
-Make sure you have [Node.js](http://nodejs.org/) installed.
+## Build instructions
```

Comment:
@mmrj I modeled this section after the changes you made to the Node.js bootstrapping readme: https://github.com/launchdarkly/hello-bootstrap/pull/10/files